app/src/forms/file/routes.js

```diff
@@ -13,7 +14,7 @@ routes.post('/', middleware.publicRateLimiter, hasFileCreate, fileUpload.upload,
   await controller.create(req, res, next);
 });
 
-routes.get('/:id', currentFileRecord, hasFilePermissions(P.SUBMISSION_READ), async (req, res, next) => {
+routes.get('/:id', middleware.publicRateLimiter, apiAccess, currentFileRecord, hasFilePermissions(P.SUBMISSION_READ), async (req, res, next) => {
```

Comment:
I'm afraid that my latest commit is going to break this - can you update your branch and use the apiKeyRateLimiter here?

Reply:
In case you don't have it: the spec is deployed at `http://localhost:8080/app/api/v1/docs`

Reply:
Thank you, I was wondering about the new rate limiting so this was really helpful! I have now updated the code and the api docs.